### ISSUE TITLE
Suggested name for code battle

### ISSUE THREAD

**Issue by VictorDenisov** [colobot/colobot#786]
How about calling code battles - colobattles? It's a little bit less straight forward, but more fun, isn't it?

**Comment by OctoNezd** [comment]
In translations it will look strange(for example:КолоБитвы) and we don`t colonize anything in Code Battles

**Comment by VictorDenisov** [comment]
Yeah, my only idea was that they are in colobot game and they have very special format which you can't see in any other game. I agree that it looks less meaningful, but makes these battles a brand not just a code battle. So, i would say it really depends on whether Terranova team believe that they invented a new type of battle(which is specific to colobot) or just a code battle format of which can vary.

**Comment by OctoNezd** [comment]
Hmm.
Following your idea, I thinked that would be great if there will be battle like "Who can make more titan/Kill more aliens". That`s will be a ColoBattle, but CodeBattle is when two players "fighting" using code.
